### ISSUE TITLE
(Option to) remove red border

### ISSUE THREAD

**Issue by samgovier** [enenumxela/vscode-signed-dark-pro#1]
**Is your feature request related to a problem? Please describe.**
This theme is practically perfect, it's just that the red border is very distracting, caused by the following JSON:
```json
        /* https://code.visualstudio.com/api/references/theme-color#window-border */
        "window.activeBorder": "#ff0000",
        "window.inactiveBorder": "#ff0000",
```

**Describe the solution you'd like**
If you'd like to remove it I wouldn't mind: otherwise, you could just provide a second theme in the bundle that removes it, eg: `signed-dark-pro-borderless.color-theme.json`

**Describe alternatives you've considered**
I do just have them commented out in my local copy of the json file. Not a big need, just wanted to request!

**Comment by enenumxela** [comment]
@samgovier Would please give me steps to reproduce? Perhaps a screenshot even?

**Comment by samgovier** [comment]
I made a pull request with the changes that also has screenshots. Feel free to change it yourself if you prefer.
#2 : [red window border removal](https://github.com/enenumxela/vscode-signed-dark-pro/pull/2) 

Are you using Linux or Mac? I'm guessing it doesn't show up on those systems: I can see it on Windows.

**Comment by enenumxela** [comment]
I'm on Linux, and it doesn't show. Thanks for the PR

**Comment by enenumxela** [comment]
I've just released a minor update, v1.1.0, check it out the close this issue if it's addressed

**Comment by samgovier** [comment]
Yep looks great. Thanks Alex!
